packages/compiler/src/compiler.ts

```diff
@@ -132,41 +145,41 @@ export class ComponentCompiler {
     trustedRoot,
   }: ParseComponentTreeParams) {
     // transpile and cache the Component
-    const transpiledComponentSource = this.getTranspiledComponentSource({
+    const {
+      children,
+      exports,
+      imports,
+      source: transpiledComponentSource,
+    } = this.getTranspiledComponentSource({
       componentPath,
       componentSource,
+      isComponentPathTrusted: isComponentPathTrusted
+        ? isComponentPathTrusted
+        : () => false,
```

Comment:
A small one, just to save a line:
```suggestion
      isComponentPathTrusted: isComponentPathTrusted || () => false,
```